stdlib/websub/src/main/ballerina/src/websub/Module.md

```diff
@@ -127,385 +104,26 @@ A hub client endpoint is also made available to publishers and subscribers to pe
 - Subscribers
   - Subscribe/Unsubscribe to/from topics at a hub
 
-## Samples
-This sample demonstrates a Subscriber Service with `subscribeOnStartUp` set to true. This will result in a
- subscription request being sent to the specified hub for the specified topic, with the specified lease seconds value 
- and the specified secret for authenticated content distribution.
-Since an `onIntentVerification` resource function is not included, intent verification for subscription and unsubscription 
-requests would happen automatically, if the topic specified in the request matches that specified as an annotation or 
-that discovered for the annotated resource URL.
- 
-```ballerina
-import ballerina/log;
-import ballerina/websub;
-
-listener websub:Listener websubEP = new(8181);
-
-@websub:SubscriberServiceConfig {
-    path: "/websub",
-    subscribeOnStartUp: true,
-    target: ["<HUB_URL>", "<TOPIC_URL>"],
-    leaseSeconds: 3600,
-    secret: "<SECRET>"
-}
-service websubSubscriber on websubEP {
-
-    resource function onNotification(websub:Notification notification) {
-        var payload = notification.getTextPayload();
-        if (payload is string) {
-            log:printInfo("WebSub Notification Received: " + payload);
-        } else {
-            log:printError("Error retrieving payload as string", payload);
-        }
-    }
-}
-```
-
-Explicit intent verification can be done by introducing an `onIntentVerification` resource function.
```

Comment:
These concepts are needed. Please undo all of these and add a minimum code snippet introducing a heading. For example, the here L164 can be replaced with a topic as follows
>Explicit intent verification
Under that keep the statement in the L164
Then keep only L179 to L189 as the code snippet which explains this further

Can you do the same for the rest as well? Please add a proper heading as a minimum code. It doesn't matter these are repeated in BBEs. These are the basics. So the user should learn this before moving to the BBEs